frontend/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```diff
@@ -274,6 +317,21 @@ export const UICustomizations = {
         return <div>{value?.length}</div>;
       }
     },
+    additionalCustomizationForMobile : (value, key, t) => {
```

Comment:
if tenantId is available in api response use that instead of doing Digit.ULBService.getCurrentTenantId